website/project/metadata/utils.py

```diff
@@ -0,0 +1,33 @@
+def serialize_meta_schema(meta_schema):
+    if not meta_schema:
+        return None
+    return {
+        'schema_name': meta_schema.name,
+        'schema_version': meta_schema.schema_version,
+        'schema': meta_schema.schema
+    }
+
+def serialize_draft_registration(draft, auth=None):
+    from website.profile.utils import serialize_user  # noqa
```

Comment:
Why are these imports inside the function? (I assume to avoid a circular import- but we shouldn't take that for granted)